LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Eddie Jaoude Community
+Copyright (c) 2021 EddieHub Community
```

Comment:
I don't think the year has to be changed though.

Reply:
> 

But this is template

Reply:
```suggestion
Copyright (c) 2020 EddieHub Community
```

Reply:
> > 
> 
> But this is template

Ooops..